monitor/monitor_load.go

```diff
@@ -1,7 +1,7 @@
 package monitor
 
 import (
-	"github.com/blockassets/bam_agent/controller"
+	"github.com/blockassets/bam_agent/service"
```

Comment:
Minor formatting. Please change the order of imports to put the github ones at the bottom.